test/mason/mason-test-exit/exitCodeTest.skipif

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+foo="$(command -v mason)"
```

Comment:
I believe this can just be:

```bash
command -v mason
status=$?
```

Reply:
I was getting behavior where just doing `command -v mason` would work when mason was not installed but would get a invalid literal error when `command v mason` would return the path. Im not entirely sure why that was the case.

Reply:
Oh, my mistake. I believe the common usage is this:

```bash
command -v mason > /dev/null
status=$?
```

Reply:
Ah, thanks. Just pushed that on